src/HDF5.jl

```diff
@@ -146,8 +146,8 @@ mutable struct File <: DataFile
         f
     end
 end
-convert(::Type{hid_t}, f::File) = f.id
-function show(io::IO, fid::File)
+Base.convert(::Type{hid_t}, f::File) = f.id
```

Comment:
If all unqualified uses have been replaced with this and similar functions, the `import Base:` entries should be removed.

Reply:
Thanks, will update the PR.